### ISSUE TITLE
ignore-same-line

### ISSUE THREAD

**Issue by aleksey-bykov** [palantir/tslint#2997]
incorrect description for the "curly" rule:

![image](https://user-images.githubusercontent.com/937933/27881546-127e4aca-6197-11e7-85fe-78c79df01119.png)

should be

![image](https://user-images.githubusercontent.com/937933/27881565-2e44d45e-6197-11e7-948f-a791bbd90034.png)

**Comment by ajafff** [comment]
you always need the boolean as the first option. Or you use the new options format. So you can choose between:

```js
"curly": [true, "ignore-same-line"]
```
```js
"curly": {
    "severity": "default",
    "options": "ignore-same-line"
}
```
```js
"curly": {
    "severity": "default",
    "options": ["ignore-same-line"]
}
```

**Comment by aleksey-bykov** [comment]
curly: [true, and
